Replace service (#770)
Esri/arcgis-runtime-samples-java: ogc/wmts-layer/README.md

## How it works
 
1. Create a `WmtsService` using the URL of the WMTS Service.
2. After loading the WMTS service, get the list of `WmtsLayerInfos` from the service info: `service.getServiceInfo().getLayerInfos()`

> 3. Use one of the layer infos to create a new `WmtsLayer(layerInfo)`

Reviewer: just to make this super clear what parameter is being passed:

```suggestion
3. Create a new `WmtsLayer` from a `WmtsLayerInfo`
```

## About the data

> We acknowledge the use of imagery provided by services from the [Global Imagery Browse Services (GIBS)](https://wiki.earthdata.nasa.gov/display/GIBS/), operated by the [NASA/GSFC/Earth Science Data and Information System (ESDIS)](https://earthdata.nasa.gov/) with funding provided by NASA/HQ.

Reviewer: We could add a bit more information here about the data (I checked this looking at the capabilities xml: https://gibs.earthdata.nasa.gov/wmts/epsg4326/best/1.0.0/WMTSCapabilities.xml):

```suggestion
We acknowledge the use of imagery provided by services from the [Global Imagery Browse Services (GIBS)](https://wiki.earthdata.nasa.gov/display/GIBS/), operated by the [NASA/GSFC/Earth Science Data and Information System (ESDIS)](https://earthdata.nasa.gov/) with funding provided by NASA/HQ. This sample shows the Digital Elevation Model (Color Index, SRTM) layer from the WMTS service provided by GIBS, using the "SRTM_Color_Index" identifier. 
```
